server/api/event/index.js

```diff
@@ -109,4 +111,37 @@ router.put('/:eventId', function(req, res) {
   });
 });
 
+// Adds a sheep and event to the Sheep Event table
+router.post('/add-participant/:eventId', function (req, res) {
+  var sheep = req.body.user;
```

Comment:
Wouldn't `req.body.user` be a string? If so, I don't think line 123 would work. Also, I think it would be better to name it `req.body.userId`.